frontend/public/components/custom-resource-definition.tsx

```diff
@@ -27,8 +36,7 @@ const namespaced = crd => crd.spec.scope === 'Namespaced';
 const CRDRow = ({obj: crd}) => <div className="row co-resource-list__item">
   <div className="col-lg-3 col-md-4 col-sm-4 col-xs-6">
     <span className="co-resource-link">
-      <ResourceIcon kind="CustomResourceDefinition" />
-      <Link className="co-resource-link__resource-name" to={`/k8s/all-namespaces/${referenceForCRD(crd)}`}>{_.get(crd, 'spec.names.kind', crd.metadata.name)}</Link>
+      <ResourceLink kind="CustomResourceDefinition" name={crd.metadata.name} namespace={crd.metadata.namespace} />
```

Comment:
We need to change this to not show metadata.name if we want to keep the same behavior. Can you try

```suggestion
      <ResourceLink kind="CustomResourceDefinition" name={crd.metadata.name} namespace={crd.metadata.namespace} displayName={_.get(crd, 'spec.names.kind'} />
```